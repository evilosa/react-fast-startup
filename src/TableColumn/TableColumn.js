import React from 'react';

const TableColumn = () => {
  return <div>Table column</div>;
};

export default TableColumn;