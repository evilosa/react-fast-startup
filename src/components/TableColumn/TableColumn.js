import React from 'react'
import PropTypes from 'prop-types'

const TableColumn = () => {
  return <div>Table column</div>
}

TableColumn.defaultProps = {
  header: '',
  propName: null,
  footer: null,
  width: '1 0',
  action: false,
}

TableColumn.propTypes = {
  header: PropTypes.string.isRequired,
  propName: PropTypes.string,
  width: PropTypes.string,
  action: PropTypes.bool,
  onClick: PropTypes.func,
}

TableColumn.displayName = 'TableColumn'

export default TableColumn