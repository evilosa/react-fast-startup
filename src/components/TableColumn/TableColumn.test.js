import React from 'react'
import { shallow } from 'enzyme'
import TableColumn from './TableColumn'

describe('TableColumn', () => {
  const component = shallow(<TableColumn/>)

  it('renders correct', () => {
    expect(component).toMatchSnapshot()
  })
})