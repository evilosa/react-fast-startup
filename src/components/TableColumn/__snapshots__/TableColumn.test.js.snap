// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableColumn renders correct 1`] = `
<div>
  Table column
</div>
`;
